src/components/panel/panel.js

```diff
@@ -757,8 +807,8 @@ MdPanelService.prototype.create = function(config) {
     return this._trackedPanels[config.id];
   }
 
-  // If no ID is set within the passed-in config, then create an arbitrary ID.
   this._config = {
+    // If no ID is set within the passed-in config, then create an arbitrary ID.
     id: config.id || 'panel_' + this._$injector.get('$mdUtil').nextUid(),
```

Comment:
This looks like it would call `$injector.get('$mdUtil')` every time somebody doesn't pass in an id. It might be more appropriate to cache it once on init.